pages/search_page.py

```diff
@@ -0,0 +1,20 @@
+from constants.search import SearchConsts
+from pages.base_page import BasePage
+from pages.header import Header
+
+
+class SearchPage(BasePage):
```

Comment:
Does search is page ? I think is a part of the Header

Reply:
Got it,  i'll fix it